app/assets/javascripts/discourse/app/components/invite-panel.js

```diff
@@ -288,6 +287,11 @@ export default Component.extend({
       : "topic.invite_reply.username_placeholder";
   },
 
+  @discourseComputed("isStaff", "siteSettings.must_approve_users")
+  showApprovalMessage(isStaff, mustApproveUsers) {
+    return isStaff && mustApproveUsers;
+  },
+
```

Comment:
Those two computed properties are good candidates for replacing with the `and()` computed helper

Reply:
Good idea, done!